chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+
+Image: https://images.app.goo.gl/J6SU6RYryXXVT9uj7
+Demo: https://huggingface.co/facebook/wav2vec2-base-960h
+
+Speech recognition is challenging as it requires joint knowledge of audio and text. The input audio might have lots of background noise, making it difficult to pick-out the spoken speech. The written text might have characters which don’t have an acoustic sound, such as punctuation, which are difficult to infer from audio alone. These are all hurdles we’ll have to tackle when building effective speech recognition systems.
+
+[//]: # Speech recognition is challenging as it requires a model to leverage knowledge about two domains concurrently: the input audio and the output text. Systems need to extract the relevant features from the input audio, possibly disentangling the spoken speech from background noise. From these features, they must then infer the contextual relation of words and sentences to form accurate and coherent sentences.
+
+Now that we’ve defined our task, we can begin looking into speech recognition in more detail. Specifically, we’ll cover:
+* How to choose a dataset
+* How to load a dataset
+* What models we can use
+* How to prepare audio-text data
+* Metrics for speech recognition
+* How to fine-tune an ASR system with the Trainer API
+
+### How to choose a dataset
+As with any machine learning problem, our model is only as good as the data that we train it on. Speech recognition datasets vary considerably in how they are curated and the domains that they cover. To pick the right dataset, we need to match our criteria with the features that a dataset offers.
+
+Before we pick a dataset, we first need to understand some of the key defining features.
+
+#### 1. Number of hours
+Simply put, the number of training hours indicates how large the dataset is. It’s analogous to the number of training examples in an NLP dataset. However, bigger datasets aren’t necessarily better. If we want a model that generalises well, we want a diverse dataset with lots of different speakers, domains and speaking styles.
+
+#### 2. Domain
+The domain entails where the data was sourced from, whether it be audiobooks, podcasts, YouTube or financial meetings. Each domain has a different distribution of data. For example, audiobooks are recorded in high-quality studio conditions (with no background noise) and text that is taken from written literature. Whereas for YouTube, the audio likely contains more background noise and a more informal style of writing.
+
+We need to match our domain to the conditions we anticipate at inference time. For instance, if we train our model on audiobooks, we can’t expect our model to perform well in noisy environments.
+
+#### 3. Speaking style
+The speaking style falls into one of two categories:
+* Narrated: read from a script
```

Comment:
For some reason, this list doesn't render correctly on the frontend - maybe try adding a new line?
```suggestion

* Narrated: read from a script
```

Reply:
Resolved in https://github.com/huggingface/course/pull/294/commits/9d111104b597c83c0eb05f25f36544eed01eb3a0